dm/ctl/master/list_member.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package master
+
+import (
+	"context"
+	"os"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/pingcap/dm/dm/ctl/common"
+	"github.com/pingcap/dm/dm/pb"
+
+	"github.com/pingcap/errors"
+	"github.com/spf13/cobra"
+)
+
+// NewListMemberCmd creates an ListMember command
+func NewListMemberCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list-member master/worker/leader",
```

Comment:
Can we list all members if no more argument specified? in other words, make `master`/`worker`/`leader` become optional arg (`[--master] [--worker] [--leader]`).